.pre-commit-config.yaml

```diff
@@ -60,31 +57,30 @@ repos:
           - '88'
           - --blank
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.6
+    rev: v0.5.0
     hooks:
       - id: ruff
       #   args:
       #     - --verbose
   - repo: https://github.com/kynan/nbstripout
-    rev: 0.6.1
+    rev: 0.7.1
     hooks:
       - id: nbstripout
         args:
           - --extra-keys
           - metadata.kernelspec metadata.language_info.version metadata.vscode
   - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.7.1
+    rev: 1.8.5
     hooks:
```

Comment:
Will be replaced entirely by `ruff` which supports jupyter notebooks and formatting since of a few versions ago. Check the [FAQ](https://docs.astral.sh/ruff/faq/).

Reply:
Great -- that will be for a new PR or do you want to add it here already?